templates/openharmony/native/engine/手动构建说明.md

```diff
@@ -0,0 +1,112 @@
+# 手动操作指南
+## 一.环境配置
+下载DevEco开发工具并配置相关环境:
```

Comment:
I think it is not needed. Cocos will release docs after integrating with editor.

Reply:
it is deleted